src/test/java/ut/org/catrobat/jira/timesheet/services/impl/TimesheetServiceImplTest.java

```diff
@@ -148,9 +147,9 @@ public void testUpdateTimesheetPractialhours() throws Exception {
 
         //Act
         final int newPracticalHours = 300;
```

Comment:
You can completely remove the `testUpdateTimesheetPractialhours()` test since this property does not exist anymore.

Reply:
done